no-tech-for-apartheid.md

```diff
@@ -6,77 +6,73 @@ permalink: /no-tech-for-apartheid/
 
-These are not individual instances. Tech is a responsible actor in the current
-crisis.
+This is a moment of crisis. **First and foremost, tech workers must organize to demand their employers cease any defense contracts.** Through the [No Tech For Apartheid](https://www.notechforapartheid.com) (NOTA) campaign, workers at Google and Amazon are [taking bold action](https://www.latimes.com/business/story/2023-08-29/google-cloud-employees-protest-israeli-military-contract) demanding the cancelation of Project Nimbus, a 1.2B dollar contract with the Israeli military and government to build cloud infrastructure for military use. Workers across the tech industry are starting to follow, organizing to demand their companies cease any defense contracts with the Israeli state.
 
-## “What can Tech do?”
+**Second, companies must stop propagating Israeli misinformation and censoring Palestinian solidarity content.** Our employers have extensive experience protecting freedom of expression, especially that of marginalized and oppressed communities, and ensuring that users have access to verified information. 
 
-As [No Tech For Apartheid](https://www.notechforapartheid.com), workers at
-Google and Amazon are taking bold action demanding the cancelation of the
-billion dollar contract with the Israeli government and military known as
-Project Nimbus. Workers across the tech industry are starting to follow,
-organizing to demand their companies cease any defense contracts with the
-Israeli state.
+**Third, tech companies must protect the welfare of Palestinian, Arab, and Muslim tech workers, and others speaking out in solidarity against apartheid.** Tech companies must start by addressing harassment and discrimination targeted at them in and out of the workplace.
 
-Second, companies can stop their censorship of Palestinian solidarity content.
+It’s time for tech companies to recognize that the labor they employ have voices, perspectives, and needs that they can no longer ignore. Technology should be used to build a better world, not wreck it, and it’s time for us to call on our employers to build it.
 
-Third, tech companies can protect the welfare of Palestinian, Arab, and Muslim
-tech workers, and others speaking in solidarity with the Palestinian cause by
-equitably addressing workplace and other harassment and discrimination.
+## **“What can I do?”**
 
-## “What can I do?”
+It is so easy to fall into despair and feel hopeless. But if we take action together, collectively workers have the power to change this situation. Here are five immediate action items you can take: 
 
-It is so easy to fall into despair and feel hopeless. But if we take action
-together, collectively workers have the power to change this situation. 
+1. ORGANIZE: Join the [No Tech For Apartheid](https://www.notechforapartheid.com) campaign. Join our upcoming mass calls to meet each other and learn about upcoming actions: 
+    * Nov 9, Thursday - 2pm pt / 5pm et 
```

Comment:
Can you please add registration links (sorry for the delay):

11/14, 8 p.m. ET / 5 p.m. PT: https://jvp-org.zoom.us/meeting/register/tZUqc-morT0jH9IuCn8LDVDlhWUquCVZFrx5#/registration 

11/16, 5 p.m. ET / 2 p.m. PT: https://jvp-org.zoom.us/meeting/register/tZEsdO2uqTotGtL1fxdK42rL5KT1lThUis_q#/registration